experimental/parser/parse_path.go

```diff
@@ -113,9 +120,13 @@ func parsePath(p *parser, c *token.Cursor) ast.Path {
 			// consume this token.
 			p.Error(errUnexpected{
 				what:  next,
-				where: taxa.QualifiedName.In(),
+				where: taxa.QualifiedName.After(),
 				want:  taxa.NewSet(taxa.Ident, taxa.Parens),
-			})
+			}).Apply(report.SuggestEdits(
+				prevSeparator,
+				fmt.Sprintf("delete the extra `%s`", prevSeparator.Text()),
```

Comment:
Deleting is unlikely correct. Maybe replace with a space? But even then, the result will most likely not be valid. I think we may be best off _not_ emitting a suggestion here since I doubt we could emit a correct suggestion or one that actually produces a valid program.

Reply:
Fair.